### PR TITLE
badge: Simplify the css selector utility naming

### PR DESCRIPTION
**Pull request by jaketrent**
An attempt to make it more readable.  See last commit only.  Merge after #829.

### CODE REVIEW

File: packages/badge/src/js/index.js [hunk 1]
```diff
@@ -1,21 +1,10 @@
...
-  return `.psds-theme--${t}`
+export function select(themeName, appearance, color) {
+  return (
+    '.psds-badge--apearance-' +
+    appearance +
+    '.psds-badge--color-' +
+    color +
+    '.psds-theme--' +
+    themeName
+  )
```
Comment (danethurber) [inline packages/badge/src/js/index.js]: i know i've built something similar in other packages. i wonder if we can find a generic/reuseable solution.

also i want to show you how i've been doing css w/cascade in gitprime's repo and see if there is a way to make it work with our repo and css build

Comment (jaketrent) [reply packages/badge/src/js/index.js]: Where's the other thing you built?  I'd be happy to consider it after seeing what was generalizable.  Though I do think this is so simple as to not need something very fancy.  I almost removed this string builder for this very reason.

Comment (EdwardIrby) [reply packages/badge/src/js/index.js]: Think this is what he's referring to https://github.com/pluralsight/design-system/blob/master/packages/emptystate/src/react/index.js#L20

Comment (jaketrent) [reply packages/badge/src/js/index.js]: The puurpose here seems slightly different than what was discussed in the PR.

- PR is creating a string combining selectors.
- The code ref’ed here is combining selectors in a glamor call.

The point of wanting generality still stands.  The PR just doesn’t seem to overlap with the purpose of the code referenced.

When I said as much in the Slack convo, I got the thumbs up from Dane, so I think we're ok to merge.

**Pull request merged**